### PR TITLE
Test Commit/Push

### PR DESCRIPTION
**Pull request by Benjamin-Earley**

### CODE REVIEW

**Review by ari7946 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -30,14 +30,27 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+    Unlike .forEach, .map returns a new array while passing those elements to the callback function.
+
 2. What is the difference between a function and a method?
+    
+    Functions can hold values inside them while methods are ususally used to invoke functions.
```
Comment (ari-mercado) [inline README.md:38]: A method is a function housed within an object

File: README.md [hunk 2]
```diff
@@ -30,14 +30,27 @@ Edit this document to include your answers after each question. Make sure to lea
...
 
 3. What is closure?
 
+    Closure refers to the lexical scope of a function.
```
Comment (ari-mercado) [inline README.md:41]: This is correct. Furthermore, A closure is a nested function that has access to its outer function's variables.  Closures store references to the outer function's variables even after function invocations have completed.

File: challenges/objects-arrays.js [hunk 3]
```diff
@@ -49,8 +71,13 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
...
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
-const universities = [];
-console.log(universities)
+let universities = [];
+for(let i = 0; i < graduates.length; i++) {
+  universities.push(graduates[i].university);
+  universities.sort();
+}
+
```
Comment (ari-mercado) [inline challenges/objects-arrays.js:79]: very good console log format

**Review by ari7946 — APPROVED** [review]
Great Job, Ben.

**Pull request closed without merging**